browser/images/dark/lc_square.svg

```diff
@@ -1 +0,0 @@
-<svg viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="m2.5 2a.50005.50005 0 0 0 -.5.5v19a.50005.50005 0 0 0 .5.5h19a.50005.50005 0 0 0 .5-.5v-19a.50005.50005 0 0 0 -.5-.5zm.5 1h18v18h-18z" fill="#fafafa"/></svg>
```

Comment:
This SVG file doesn't appear to be mentioned in the aliases anywhere. Was it meant to be deleted?